tests/serverpod_test_server/lib/src/endpoints/json_protocol.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:async';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+
+class JsonProtocolEndpoint extends Endpoint {
+  Future<ScopeServerOnlyField> getJsonForProtocol(
```

Comment:
Suggestion(NIT): Rename to `getScopeServerOnlyField`.

This better describes what we are getting. The `jsonForProtocol` is an implementation detail that doesn't need to be reflected in the method.